### ISSUE TITLE
multi replys

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#66]
```
i don't know if the summary's clear enough, so here we go... from time to 
time i need to reply to more than one person in a single tweet. when i click 
in the button reply the nickname appears in the textbox so i don't have to 
memoriza it... but when i want to include another person y have to copy the 
nick name. Would be handy that when clicking the button to reply, if some 
text is already written in the textbox, the nick gets appended, and does not 
override the previous text. I hope i have shown my point :)
```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 2 Feb 2010 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
In older versions you could click on the nickname for this. I will fix this bug 
in the 
next version.
```

Original comment by `tro...@gmail.com` on 3 Feb 2010 at 3:56
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
nice, thank you! Honestly i didn't notice that :(
```

Original comment by `mich.ba...@gmail.com` on 3 Feb 2010 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
Closed by 2d139ba5b159 revision.
```

Original comment by `tro...@gmail.com` on 10 Feb 2010 at 11:30
- Changed state: **Fixed**
